include/version.h

```diff
@@ -27,6 +27,7 @@ namespace kiwix
 {
   typedef std::vector<std::pair<std::string, std::string>> LibVersions;
   LibVersions getVersions();
+  std::string getVersionsString();
```

Comment:
What about slightly enhancing the `printVersions()` function instead?

```
void printVersions(std::ostream& out = std::cout);
```

Then `getVersionsString()` can be replaced with

```
std::ostringstream oss;
zim::printVersions(oss);
kiwix::printVersions(oss);
std::string versionsString = oss.str();
```

Reply:
Fixes in last commit